airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -696,10 +698,11 @@ def cleanup(self, pod: k8s.V1Pod, remote_pod: k8s.V1Pod):
                         f"Pod {pod and pod.metadata.name} returned exit code "
                         f"{self.skip_on_exit_code}. Skipping."
                     )
-            raise AirflowException(
-                f"Pod {pod and pod.metadata.name} returned a failure:\n{error_message}\n"
-                f"remote_pod: {remote_pod}"
-            )
+            raise AirflowException('\n'.join(filter(None, [
+                f"Pod {pod and pod.metadata.name} returned a failure.",
+                error_message,
```

Comment:
I think a new line to separate this would be nicer. Also, previously if the value was not set, we were converting it to an empty string. Now, we're converting it to an empty string, would it have some impact?

Reply:
@pankajkoti the join symbol is `\n` so that all messages will be logged in new line.
Now if the value is not set it will be filtered out from exception message, so that same behaviour will be achieved but without explicit set to an empty string

Reply:
okay, understood, thanks!